src/yb/consensus/metadata.proto

```diff
@@ -139,6 +139,20 @@ message ConsensusStatePB {
   required RaftConfigPB config = 3;
 }
 
+message ListAllOpIdPB {
+  // Operation ID for the latest request received.
+  optional int64 last_received_opid_index = 1;
```

Comment:
Instead of op index (int64), OpIdPB (term, index) looks more clear. As working on this item, let's make the debug info as detailed as possible.